libs/image/include/image/ColorTransform.h

```diff
@@ -231,8 +230,7 @@ inline std::unique_ptr<uint8_t[]> fromLinearToRGB_10_11_11_REV(const LinearImage
     using namespace filament::math;
     size_t w = image.getWidth();
     size_t h = image.getHeight();
-    UTILS_UNUSED_IN_RELEASE size_t channels = image.getChannels();
```

Comment:
Does MSVC support  [`[[maybe unused]]`](https://en.cppreference.com/w/cpp/language/attributes/maybe_unused)? I don't think MSVC supports `__has_attribute` so we might have to define `UTILS_UNUSED_IN_RELEASE` specifically for MSVC.

Reply:
[It does](https://docs.microsoft.com/en-us/cpp/cpp/attributes?view=msvc-170) since VS 15.3 (and only if /std:c++17 (or later) compiler flag is used). This can be checked via [predefined macros](https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170) of MSVC.
`_MSC_VER` can effectively identify the VS version (the corresponding value to VS 15.3 [is 1911](https://dev.to/yumetodo/list-of-mscver-and-mscfullver-8nd)) and the language version compiler flag can be tested via `_MSVC_LANG`.
See ac08c4018c9bfec4285ce2e084f4326b706631fe.

Reply:
Note that now `UTILS_UNUSED_IN_RELEASE` is back in ColorTransform.h it could still emit a warning for someone who uses a very old VS version or compiles Filament with /std:c++14 or lesser (although I am not even sure if Filament would compile that way)

Reply:
Thanks! Older versions of MSVC I'm not too concerned about; like you said, not even sure if they would compile.